common/djangoapps/mailchimp_pipeline/signals/handlers.py

```diff
@@ -36,41 +36,44 @@ def update_mailchimp(email, data):
 @receiver(post_save, sender=EmailPreference)
 def sync_email_preference_with_mailchimp(sender, instance, **kwargs):
     email_preferences = instance
+    opt_in = ''
```

Comment:
'yes' and 'no' translate to 'TRUE' and 'FALSE' respectively for email opt_in preference on mailchimp. Empty opt_in gets set as empty on mailchimp. New regflow does not have any flow that'd make it possible for opt_in to be empty, but I won't be changing this logic because some other instance of code might be sending empty opt_in